### PR TITLE
Update Using Grayskull Add version 2.10.0

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@bollwyvl - removed noarch !, Thank you for your review and assistance. the build is passing now.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,57 +1,43 @@
...
-{% set cov_threshold = 57 %}  # [osx]
-{% set cov_threshold = 0 %}  # [win]
-
 test:
-  source_files: src/tests
   imports:
     - time_machine
   commands:
     - pip check
-    # can't test on windows due to use of ``time.tzset``
```
Comment (bollwyvl) [inline recipe/meta.yaml:41]: the tests help keep everything running smoothly at updates, especially for compiled works. please restore.

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@bollwyvl ready for review again. Thank you.

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Pull request merged**